test/testunusedvar.cpp

```diff
@@ -2010,6 +2011,16 @@ class TestUnusedVar : public TestFixture {
         ASSERT_EQUALS("[test.cpp:2:18]: (style) struct member 'S::a' is never used. [unusedStructMember]\n", errout_str());
     }
 
+    void structmember29() { // #14130
+        checkStructMemberUsage("struct S\n"
+                               "{\n"
+                               "private:\n"
```

Comment:
We do not need the `private:` here. I included that in the example as to show that the unused private member Clang compiler warnings are being suppressed by the attribute.